NEWS.md

```diff
@@ -1,3 +1,6 @@
+## 2024-03-XX 2.2.2
```

Comment:
It should be 2.2.0-SNAPSHOT iteration, points from 2.2.1 should also move under 2.2.0-SHAPSHOT